crowdsourcing/serializers/file.py

```diff
@@ -0,0 +1,17 @@
+from crowdsourcing import models
+from rest_framework import serializers
+from crowdsourcing.models import File
+from crowdsourcing.serializers.file import File
+from crowdsourcing.serializers.dynamic import DynamicFieldsModelSerializer
+
+class FileSerializer(DynamicFieldsModelSerializer):
```

Comment:
hyperlinkedserializer would be the best here in case you need to fetch the file later but this will work for now since we dont need to download the file